assets/src/edit-story/components/inspector/karma/inspectorTabs.karma.js

```diff
@@ -144,7 +144,7 @@ describe('Inspector Tabs integration', () => {
         name: /^Review Checklist$/,
       });
       await fixture.events.click(reviewButton);
-      await fixture.events.sleep(300);
+      // await fixture.events.sleep(300);
```

Comment:
Should we remove this?

Reply:
i will :) i'm rerunning the test suites soooo many times just to make sure things work. these e2e tests are kicking my butt. 

thank you for flagging this. i promise not to merge it in.